vgen/src/main/java/io/gapi/vgen/py/PythonLanguageProvider.java

```diff
@@ -58,9 +58,10 @@
 
     Doc filenameDoc = snippets.generateFilename(element);
     String outputFilename = filenameDoc.prettyPrint();
+    Doc body = snippets.generateBody(element);
     List<String> importList = importHandler.calculateImports();
     // Generate result.
-    Doc result = snippets.generateClass(element, importList);
+    Doc result = snippets.generateModule(element, body, importList);
```

Comment:
Sorry, I haven't actually read through this whole change, but it looks like right here it's still the old flow of calling "calculateImports()" first, and then passing this list to the snippet entry point?